constantinCurtis/src/main/java/generics/second/task/Main.java

```diff
@@ -0,0 +1,83 @@
+package generics.second.task;
```

Comment:
if you want to sort your tasks adjust paths please, could be generics/task/(first/second)

Reply:
Changed, thanks!